CefSharp.OutOfProcess.WinForms/ChromiumWebBrowser.cs

```diff
@@ -64,13 +70,15 @@ public class ChromiumWebBrowser : Control, IChromiumWebBrowserInternal
         /// </summary>
         /// <param name="host">Out of process host</param>
         /// <param name="initialAddress">address that will be initially loaded in the browser</param>
-        public ChromiumWebBrowser(OutOfProcessHost host, string initialAddress)
+        /// <param name="requestContextPreferences">requestContextPreferences to set</param>
```

Comment:
The comment needs to be updated to describe the actual behaviour, a new `RequestContext` sharing the same settings as the `Global`.